add_gem_package.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 usage() {
-	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR [REPO]]]"
+	echo "Usage: $0 GEM_NAME [TEMPLATE [PACKAGE_SUBDIR]]"
 	echo "Valid templates: $(ls gem2rpm | sed 's/.spec.erb//' | tr '\n' ' ')"
 	echo "Valid repos: foreman-el8 foreman-plugins-el8 katello-el8 foreman-client-el8"
```

Comment:
This line needs to go.

Reply:
Argh, I read over it a few times and first wanted to drop the actual usage. Then forgot it.